docs/samples/README.md

```diff
@@ -1,61 +1,99 @@
+| Deploy Tensorflow Model on TFServing  | [Tensorflow SavedModel](https://www.tensorflow.org/guide/saved_model) | :heavy_check_mark: | :heavy_check_mark: | [Tensorflow Flowers](./tensorflow)  |
+| Deploy ONNX Model on ONNXRuntime  | [Exported onnx model(model.onnx)](https://github.com/onnx/tutorials#converting-to-onnx-format) | :heavy_check_mark: | :heavy_check_mark: |[ONNX Style Model](./onnx)  |
+| Deploy Model on Triton Server | [Tensorflow,PyTorch,ONNX,TensorRT](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/model_repository.html)| :heavy_check_mark: | :heavy_check_mark: | [Simple String](./triton/simple_string) |
+| Deploy model on custom KFServer | | :heavy_check_mark: | V2 | [Custom KFServer](./custom/kfserving-custom-model)|
+| Deploy model on BentoML | | :heavy_check_mark: | - | [SKLearn Iris with BentoML](./bentoml)|
+| Deploy model on custom HTTP Server | |:heavy_check_mark: | - | [Prebuilt model server](./custom/prebuilt-image)|
+| Deploy model on custom gRPC Server | |  -  | :heavy_check_mark: | [Prebuilt gRPC server](./custom/grpc-server)|
+
+In addition to deploy InferenceService with HTTP/gRPC endpoint, you can also deploy InferenceService with [Knative Event Sources](https://knative.dev/docs/eventing/sources/index.html) such as Kafka
+, you can find an example [here](./kafka) which shows how to build an async inference pipeline. 
+
+### Deploy InferenceService with Transformer
+KFServing transformer enables users to define a pre/post processing step before the prediction and explanation workflow.
+KFServing transformer runs as a separate microservice and can work with any type of pre-packaged model server, it can also 
+scale differently from the predictor if your transformer is CPU bound while predictor requires running on GPU. 
+
+| Features  | Examples |
+| ------------- | ------------- |
+| Deploy Transformer with KFServer | [Image Transformer with PyTorch KFServer](./transformer/image_transformer)  |
+| Deploy Transformer with Triton Server | [BERT Model with tokenizer](./triton/bert)  |
+
+### Deploy InferenceService with Explainer and Outlier/Drift Detector
+Model explainability answers the question: "Why did my model make this prediction" for a given instance. KFServing 
+integrates with [Alibi Explainer](https://github.com/SeldonIO/alibi) which implements a black-box algorithm by generating a lot of similar looking intances 
+for a given instance and send out to the model server to produce an explanation.
+Also in order to trust and reliably act on model predictions, it is crucial to monitor the distribution of the incoming
+requests via various different type of detectors. [Alibi Detect](https://github.com/SeldonIO/alibi-detect) checks when the distribution of incoming requests 
+is diverging from a reference distribution such as that of the training data.
```

Comment:
I think drift and outlier detection should be in a separate section. Also your explanation for them seems to focus on drift not outlier detection which is for individual requests. Maybe reword?